pkg/config/config_test.go

```diff
@@ -320,6 +320,47 @@ func TestValidateConfig(t *testing.T) {
 			},
 			expectedErr: "invalid stream idle timeout",
 		},
+		"valid id mapping": {
+			config: &PluginConfig{
+				ContainerdConfig: ContainerdConfig{
+					DefaultRuntimeName: RuntimeDefault,
+					Runtimes: map[string]Runtime{
+						RuntimeDefault: {
+							Type: plugin.RuntimeLinuxV1,
+						},
+					},
+				},
+				NodeWideUIDMapping: LinuxIDMapping{0, 800000, 65536},
+				NodeWideGIDMapping: LinuxIDMapping{0, 800000, 65536},
+			},
+			expected: &PluginConfig{
+				ContainerdConfig: ContainerdConfig{
+					DefaultRuntimeName: RuntimeDefault,
+					Runtimes: map[string]Runtime{
+						RuntimeDefault: {
+							Type: plugin.RuntimeLinuxV1,
+						},
+					},
+				},
+				NodeWideUIDMapping: LinuxIDMapping{0, 800000, 65536},
+				NodeWideGIDMapping: LinuxIDMapping{0, 800000, 65536},
+			},
+		},
+		"invalid id mapping": {
+			config: &PluginConfig{
+				ContainerdConfig: ContainerdConfig{
+					DefaultRuntimeName: RuntimeDefault,
+					Runtimes: map[string]Runtime{
+						RuntimeDefault: {
+							Type: plugin.RuntimeLinuxV1,
+						},
+					},
+				},
+				NodeWideUIDMapping: LinuxIDMapping{1, 100000, 65536},
+				NodeWideGIDMapping: LinuxIDMapping{1, 100000, 65536},
+			},
+			expectedErr: "missing root id in container",
+		},
```

Comment:
I implemented a test for different uid/gid mapping in https://github.com/kinvolk/containerd-cri/commit/76ade408a665afacb92b25c28407f22d35aa7ff2, feel free to cherry-pick if you think that's good enough.